gui/main.js

```diff
@@ -95,7 +95,8 @@ const configDefaultSettings =
                   "shared_folder_path" : ".", \
                   "enable_request_profile" : "True", \
                   "enable_request_shared_folder" : "False", \
-                  "tx_delay" : 0 \
+                  "tx_delay" : 0, \
+                  "auto_start": 0 \
```

Comment:
Suggestion: renaming this to hamlib_auto_start or so, so it's clear for what we're using this setting